gpii/node_modules/matchMakerFramework/src/MatchMakerCapabilities.json

```diff
@@ -0,0 +1,19 @@
+[
+	{
+		"id": "flat",
+		"conditions": {
+			"supported": false
+		}
+	},
+	{
+		"id": "statistical",
+		"preferences": {
+			"supported" : []
+		},
+		"metrics": {
+			"baseValue": -10.0,
+			"forEachPreference": 1.0,
+			"forEachUnsupportedPreference": 0.0
+		}
+	}
```

Comment:
Also - do you have description of the above capabilities metrics available anywhere -- I think I get the general idea, but am not certain